Dokumentationen/test/test_cases.adoc

```diff
@@ -378,12 +378,16 @@ a| *
 |===
 
 |*Beschreibung* |
+Die Werte der Voreinstellungen (Studiengang, Uhrzeit, Datum) werden aus einem festgelegten Wertebereich gewählt. Realisiert wird dies durch Dropdown-Menüs und einen Datepicker. Offensichtlich wird also erwartet, dass die Voreinstellungen nicht durch Freitextfelder erfasst werden können. (manueller Test)
 
 |*Vorbedingungen und benötigte Daten*
-a| * 
+a| * Dozent ist angemeldet 
+* Buchung wurde gestartet 
+* auswählbare Daten bezüglich Datum, Uhrzeit, Studiengang
```

Comment:
auch KW und Semester kann vom Admin in Dozentenwoche bzw Statistik ausgewählt werden

Reply:
Stimmt, daran habe ich gar nicht gedacht!